### ISSUE TITLE
questions about quora data split

### ISSUE THREAD

**Issue by ZizhenWang** [zhiguowang/BiMPM#59]
Hi, thanks for your excellent work!
Your split for quora data is used for so many succeed works, and now I want to use it. But I find the pairID in [your data](https://drive.google.com/file/d/0B0PlTAo--BnaQWlsZl9FZ3l1c28/view?usp=sharing) is not all matched to raw data successfuly. Here are many examples:

`id:  184949
raw left:  Which is the best phone under 15K in India?
raw right:  Which is the best mobile phone under 15K in India?
your q1:  How is the furniture industry ?
your q2:  What is the furniture industry like ?`
`id:  85926
raw left:  How I can enhance my English language?
raw right:  How can I increase my English fluency?
your q1:  Why is the Economist banned in China ?
your q2:  Why is Economist banned in China ?`

`id:  131673
raw left:  I am in Std 10th, scored really bad in SA1 . the marks in 5 subjects are (out of 90) 75,69,72,83,55.is there any chance of scoring 10 CGPA now?
raw right:  I scored really bad marks in 10th (CBSE) Term 1. Is there any chance of performing well by scoring well in term 2?
your q1:  By what peaceful , legal scenarios could Trump be removed from office ?
your q2:  Can Trump be impeded from taking office ?`

**Comment by Deep1994** [comment]
Yeah, I meet the same problem as you, how do you solve this issue?
